packages/scripts/README.md

```diff
@@ -25,10 +25,10 @@ _Example:_
 	"scripts": {
 		"build": "wp-scripts build",
 		"check-engines": "wp-scripts check-engines",
-		"check-licenses": "wp-scripts check-licenses --production",
```

Comment:
Could you explain this change?

Reply:
It validates all dependencies installed in the project, not only those used for production. We check regular and dev dependencies in Gutenberg, so I thought we should recommend both for other projects as well.